test/e2e/specs/blocks/list.test.js

```diff
@@ -9,6 +9,7 @@ import {
 	convertBlock,
 	pressWithModifier,
 	insertBlock,
+	META_KEY,
```

Comment:
It needs to be rebased with the `master` branch.

With the latest refactoring, we are able to reuse logic from `@wordpress/keycodeds` package and express this using `primary` modifier:

```js
await pressWithModifier( 'primary`, 'z' );
```